src/15_classes.py

```diff
@@ -1,29 +1,74 @@
 # Make a class LatLon that can be passed parameters `lat` and `lon` to the
 # constructor
 
-# YOUR CODE HERE
+
+class LatLon:
+    lat = 0
+    lon = 0
+
+    def __init__(self, lat, lon):
```

Comment:
it might be better to add the default values while being passed in to the __init__ dunder. 
so ``` def __init___(self, lat=0, lon=0): ```